src/Lecture2.hs

```diff
@@ -181,7 +259,10 @@ False
 True
 -}
 isIncreasing :: [Int] -> Bool
-isIncreasing = error "TODO"
+isIncreasing list = case list of
+    (x:y:_) | x > y -> False
+    (_:y:ys) -> isIncreasing (y:ys)
```

Comment:
You can implement `isIncreasing` in a slightly more elegant way by relying on laziness in Haskell 🦥 

```suggestion
    x : y : xs -> x < y && isIncreasing (y : ys)
```